embed.go

```diff
@@ -0,0 +1,289 @@
+
+	// to avoid a lot of boiler plate we use struct tags to map the file names
+	// this isn't strictly necessary, and we could just perpetually add to a list here
+	// but this would result in more "shot gun surgery" every time a new tx is added
+	// tradeoff is legibility vs ease of maintenance
+	// another alternative that would likely be superior is to autogenerate the code
+	// by crawling the file tree
+	scriptType := reflect.TypeOf(scripts)
+	scriptVal := reflect.ValueOf(&scripts)
+	for i := 0; i < scriptType.NumField(); i++ {
+		f := scriptType.Field(i)
+		path := f.Tag.Get("src")
+		script, err := scriptProcessor.Process(ctx, conf, path)
+		if err != nil {
+			return Scripts{}, err
+		}
+		scriptVal.Elem().Field(i).SetBytes(script)
+	}
+
+	return scripts, nil
+}
+
+type ScriptProcessor struct {
+	*project.ImportReplacer
+}
+
+// ConfigFromEnv will return config appropriate for the network
+func ConfigForEnv(network string) (Config, error) {
+	loader := config.NewLoader(&embedReaderWriter{configFS})
+	loader.AddConfigParser(json.NewParser())
+	flowConf, err := loader.Load([]string{"flow.json"})
+	if err != nil {
+		return Config{}, err
+	}
+
+	allDayAddress, err := contractAddressHex(flowConf, "AllDay", network)
+	if err != nil {
+		return Config{}, err
+	}
+	nftAddress, err := contractAddressHex(flowConf, "NonFungibleToken", network)
+	if err != nil {
+		return Config{}, err
+	}
+	ftAddress, err := contractAddressHex(flowConf, "FungibleToken", network)
+	if err != nil {
+		return Config{}, err
+	}
+	//dapperUtilityCoinAddress, err := contractAddressHex(flowConf, "DapperUtilityCoin", network)
+	//if err != nil {
+	//	return Config{}, err
+	//}
+	metadataViewsAddress, err := contractAddressHex(flowConf, "MetadataViews", network)
+	if err != nil {
+		return Config{}, err
+	}
+	viewResolverAddress, err := contractAddressHex(flowConf, "ViewResolver", network)
+	if err != nil {
+		return Config{}, err
+	}
+	burnerAddress, err := contractAddressHex(flowConf, "Burner", network)
+	if err != nil {
+		return Config{}, err
+	}
+	iPackNftContractAddress, err := contractAddressHex(flowConf, "IPackNFT", network)
+	if err != nil {
+		return Config{}, err
+	}
+	packNftContractAddress, err := contractAddressHex(flowConf, "PackNFT", network)
+	if err != nil {
+		return Config{}, err
+	}
+
+	return Config{
+		// if we have other addresses to be pulled from config, add them here
+		AllDayAddress:                allDayAddress,
+		NFTAddress:                   nftAddress,
+		FungibleTokenContractAddress: ftAddress,
+		//DapperUtilityCoinContractAddress: dapperUtilityCoinAddress,
+		MetadataViewsAddress:    metadataViewsAddress,
+		ViewResolverAddress:     viewResolverAddress,
+		BurnerAddress:           burnerAddress,
+		IPackNFTContractAddress: iPackNftContractAddress,
+		PackNFTAddress:          packNftContractAddress,
+	}, nil
+}
+
+func contractAddressHex(flowConf *config.Config, name, network string) (string, error) {
```

Comment:
We don't need these `script` or `transaction` processor. These script/transaction processors are overly complicated. Can you sync tomorrow

Reply:
Yes, let's sync tomorrow. I also thought it was way too complicated